pyomo/contrib/doe/doe.py

```diff
@@ -150,8 +149,7 @@ def stochastic_program(self,  design_values, if_optimize=True, objective_option=
         Parameters
         -----------
         design_values:
-            a ``dict`` where keys are design variable names, values are a dict whose keys are time point
-            and values are the design variable value at that time point
+            designVariable object
```

Comment:
Is passing in design_values here needed? Aren't the design_vars (an instance of the DesignVariables class) already passed into the function? What is the purpose of passing this in again? Let's discuss.